src/coreclr/src/gc/gcpriv.h

```diff
@@ -3982,6 +3997,16 @@ class gc_heap
     PER_HEAP
     alloc_list gen2_alloc_list[NUM_GEN2_ALIST-1];
 
+// TODO: using same numbers as for LOH. May need to tune this (see: https://github.com/dotnet/runtime/issues/13739)
```

Comment:
>// TODO: using same numbers as for LOH. May need to tune  [](start = 0, length = 57)

again, this will need to be tuned so I would not say "May need to tune"